nyancat/reward-claims/sig-files/chris-remus-81D443FA08A4A926.txt

```diff
@@ -0,0 +1,16 @@
+-----BEGIN PGP SIGNED MESSAGE-----
+Hash: SHA512
+
+iva1tsjrct9p7z2znsu4ehs69w5ydu5d5mu4sxst73
```

Comment:
Hi, please upload your wallet account address here to receive the rewards, not validator address :-)